corehq/apps/app_manager/templates/app_manager/v1/form_designer.html

```diff
@@ -17,6 +17,9 @@
 {% block js %}{{ block.super }}
     <script src="{% static 'moment/moment.js' %}"></script>
     <script src="{% static 'requirejs/require.js' %}"></script>
+    {% if not request|toggle_enabled:'VELLUM_BETA' %}
```

Comment:
It's safe, none of the parents compress - isn't it actually impossible to put an inheritable block inside of a compress?